pandas/tests/indexes/test_multi.py

```diff
@@ -97,6 +97,10 @@ def test_repeat(self):
             numbers, names.repeat(reps)], names=names)
         tm.assert_index_equal(m.repeat(reps), expected)
 
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False):
```

Comment:
I think you can removed all other occurences of `check_stacklevel=False` in this PR as well